libraries/entities/src/EntityScriptingInterface.h

```diff
@@ -2399,7 +2399,9 @@ public slots:
 
 
     /**jsdoc
-     * Triggered when an avatar enters an entity, but only if the entity has an entity method exposed for this event.
+     * Triggered when an avatar enters an entity, but only if the entity has an entity method exposed for this event. 
```

Comment:
```suggestion
     * Triggered when an avatar enters an entity. 
```
The "but only if" text isn't needed because it's use is explained in the two "see also" links included in the description. Or if you think it needs expanding, then it needs expanding for all entity method signals.

Reply:
I can remove that, but there is the same "but only if" for "leaveEntity"
Should I remove it there too? (see Line 2413)

Reply:
Clearly. After second thought. I went with removing the 2 "but only if..." (line 2402 and 2413)